src/tests/commonTests.ts

```diff
@@ -220,3 +221,108 @@ export async function slots(componentTagOrHTML: TagOrHTML, slots: Record<string,
 
   expect(allSlotsAssigned).toBe(true);
 }
+
+async function assertLabelable({
+  html,
+  componentTag,
+  propertyToToggle,
+  focusTargetSelector = componentTag,
+  shadowFocusTargetSelector
+}: {
+  html: string;
+  componentTag: string;
+  propertyToToggle?: string;
+  focusTargetSelector?: string;
+  shadowFocusTargetSelector?: string;
+}): Promise<void> {
+  const page: E2EPage = await newE2EPage({ html });
+  await page.waitForChanges();
+  let component: E2EElement;
+  let initialPropertyValue: boolean;
+
+  if (propertyToToggle) {
+    component = await page.find(componentTag);
+    initialPropertyValue = await component.getProperty(propertyToToggle);
+  }
+
+  const label = await page.find("calcite-label");
+  await label.callMethod("click"); // we call the method to avoid clicking the child element
+  await page.waitForChanges();
+
+  console.log(html, propertyToToggle, focusTargetSelector);
```

Comment:
got some left over logs in this file.

Reply:
This is what happens when you fix locally, but don't push. 😂